Tidy up documentation (#1528)
yewstack/yew: docs/more/testing.md

---

&lt;TODO&gt;
:::info
Yew is currently working on making it easy to test components, but this is currently a work in
progress.

> Support for [shallow rendering](https://github.com/yewstack/yew/issues/1413) and a proposal to
> [expose the code Yew uses internally for testing components](https://github.com/yewstack/yew/issues/1413)
> can be found on our Github repository.

Reviewer: I think it should be "**on** GitHub" but "**in** a repository".
```suggestion
Support for [shallow rendering](https://github.com/yewstack/yew/issues/1413) and a proposal to
[expose the code Yew uses internally for testing components](https://github.com/yewstack/yew/issues/1413)
can be found in the GitHub repository.
```
